docs/README.Windows.md

```diff
@@ -150,6 +150,7 @@ Build FFmpeg:
 ```
 make-mingwlibs.bat
 ```
+**HINT:** If you get the error `ERROR! Could not find vcvarsall.bat` you have not installed `Desktop development with C++` as a module within Visual Studio, go back and finish that step first.
```

Comment:
Using a regular command prompt instead of a developer command prompt will cause the same error, if you don't mind adding it.